lib/evss/pciu_address/configuration.rb

```diff
@@ -2,7 +2,7 @@
 module EVSS
   module PCIUAddress
     class Configuration < EVSS::Configuration
-      DEFAULT_TIMEOUT = Settings.evss.pciu_address.timeout
+      self.read_timeout = Settings.evss.pciu_address.timeout || 30
```

Comment:
Is there a risk that Settings would not have the appropriate setting? Also doesn't request options just use whatever you're specified as DEFAULT_TIMEOUT?

Reply:
Request_options uses `self.read_timeout`, before this PR the letters connection wasn't actually using `request_options` but a custom Faraday setting that wasn't picking up the overridden constant because it wasn't using `self.class::DEFAULT_TIMEOUT`.